### PR TITLE
feat: Bypass `cft test lint` rules

### PR DESCRIPTION
**Pull request by qz267**
Add `BLUEPRINT_LINT_DISABLE` env var to bypass `cft test lint` rules when set to 1.

### CODE REVIEW

File: cli/bptest/lint_interface_test.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import (
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/bpmetadata"
+	"github.com/stretchr/testify/assert"
+)
+
+type MockLintRule struct {
```
Comment (bharathkkb) [inline cli/bptest/lint_interface_test.go:12]:
```suggestion
type mockLintRule struct {
```

File: cli/bptest/lint_interface.go [hunk 2]
```diff
@@ -30,6 +32,11 @@ func (r *lintRunner) RegisterRule(rule lintRule) {
 // Run runs all the registered rules on the provided context.
 func (r *lintRunner) Run(ctx lintContext) []error {
 	var errs []error
+	if os.Getenv("BLUEPRINT_LINT_DISABLE") == "1" {
```
Comment (bharathkkb) [inline cli/bptest/lint_interface.go:35]: nit: Let's move this to a const

**Review by bharathkkb — APPROVED** [review]
Thanks @qz267!

File: cli/bptest/lint_interface_test.go (hunk 1 above)
Comment (qz267) [reply cli/bptest/lint_interface_test.go]: Done

File: cli/bptest/lint_interface.go (hunk 2 above)
Comment (qz267) [reply cli/bptest/lint_interface.go:35]: Done

**Pull request merged**